entc/gen/template/builder/query.tmpl

```diff
@@ -124,8 +124,8 @@ func ({{ $receiver }} *{{ $builder }}) FirstX(ctx context.Context) *{{ $.Name }}
 	// FirstID returns the first {{ $.Name }} ID from the query.
 	// Returns a *NotFoundError when no {{ $.Name }} ID was found.
 	func ({{ $receiver }} *{{ $builder }}) FirstID(ctx context.Context) (id {{ $.ID.Type }}, err error) {
-		var ids []{{ $.ID.Type }}
-		if ids, err = {{ $receiver }}.Limit(1).IDs(ctx); err != nil {
+		ids, err := {{ $receiver }}.Limit(1).IDs(ctx)
+		if err != nil {
```

Comment:
Why it's better/preferred? I find it less "obvious" than the current one.

Reply:
Mb you right. I close this